lib/omise/res/obj/OmiseObject.php

```diff
@@ -1,83 +1,104 @@
 <?php
 
-class OmiseObject implements ArrayAccess, Iterator, Countable {
-  // Store the attributes of the object.
-  protected $_values = array();
-
-  // Omise secret et public keys.
-  protected $_secretkey, $_publickey;
-
-  /**
-   * Setup the Omise object. If no secret and public are passed the one defined
-   * in config.php will be used.
-   * @param string $publickey
-   * @param string $secretkey
-   */
-  protected function __construct($publickey = null, $secretkey = null) {
-    if($publickey !== null) {
-      $this->_publickey = $publickey;
-    } else {
-      $this->_publickey = OMISE_PUBLIC_KEY;
+class OmiseObject implements ArrayAccess, Iterator, Countable
+{
+    // Store the attributes of the object.
+    protected $_values = array();
+
+    // Omise secret key.
+    protected $_secretkey;
+
+    // Omise public key.
+    protected $_publickey;
+
+    /**
+     * Setup the Omise object. If no secret and public are passed the one defined
+     * in config.php will be used.
```

Comment:
@bank32 to follow code comment style, do you mind if I'm not gonna change it.
like, https://github.com/omise/omise-php/pull/37/files#diff-7a0eacc7ad276bf5e64dca38d6b295b6R10